astronomer/providers/google/cloud/operators/bigquery.py

```diff
@@ -72,6 +72,10 @@ class BigQueryInsertJobOperatorAsync(BigQueryInsertJobOperator, BaseOperator):
     :param poll_interval: polling period in seconds to check for the status of job. Defaults to 4 seconds.
     """
 
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
@pankajastro All of these operator uses the same trigger?

Reply:
No, but we introduced dynamic poll_interval in these all operator/trigger

Reply:
I think we should add tests for these operators, just to make there is not regression. WDYT?

Reply:
we need to test for old version as well

Reply:
what about the test result for `apache-airflow-providers-google` < 8.11.0

Reply:
unfortunately, we run tests against only one version of oss provider maybe @phanikumv if you want we can try run for more version in a separate PR

Reply:
yeah just want to make sure that we don't break things this time

Reply:
locally you could just pin the provider to 8.11.0 and give the DAG a run, right? 

Also, In the description, I see 8.12.0 and 9.0.0, but anything >=8.12 will work with 1.15.3, we need to test one for <8.12. Can we update the description snapshot for the old release to reflect 8.11.0 instead of 9.0.0